client/lcd/swagger-ui/swagger.yaml

```diff
@@ -334,6 +334,39 @@ paths:
           description: The Tx was malformated
         500:
           description: Server internal error
+  /tx/encode:
+    post:
+      tags:
+        - ICS20
```

Comment:
Is this from a copy pasta?

Reply:
ICS20 seemed the most reasonable since it's related to creating and broadcasting transactions.